### PR TITLE
@comet/create-app: Unify console log messages

### PR DESCRIPTION
**Pull request by max-debug022**
This PR adds uniform console output for regular and verbose output:
- Warning messages are consistently displayed in yellow.
- Error messages are consistently shown in red, and the command is terminated with an exit code not equal to 1.
- Regular logs no longer utilize kleur.white since it is displayed as light grey and may be confused with verbose output.
- Verbose output is now consistently displayed in grey.

### CODE REVIEW

File: create-app/src/index.ts [hunk 1]
```diff
@@ -27,30 +27,30 @@ void (async () => {
             if (isValidProjectName(projectName)) {
                 createApp({ projectName, verbose: options.verbose, install: options.install });
             } else {
-                program.error("Please provide a valid project name.");
+                program.error("Error: Please provide a valid project name.");
```
Comment (johnnyomair) [inline create-app/src/index.ts:30]: Are these "Error:", "Info:" etc. really necessary? Isn't that what you're using the colors for?

Comment (max-debug022) [reply create-app/src/index.ts:30]: No, you are right - this was just a personal preference. Should I change it?

Comment (johnnyomair) [reply create-app/src/index.ts:30]: One could argue that this is better for accessibility however 🤔

Comment (nsams) [reply create-app/src/index.ts:30]: most cli commands don't have this prefix, even for errors.

Comment (max-debug022) [reply create-app/src/index.ts]: Removed in: https://github.com/vivid-planet/comet-starter/pull/196/commits/03056f26fef9362172218bced429d6ea13bd84cc

File: create-app/src/scripts/create-app/createApp.ts [hunk 2]
```diff
@@ -18,40 +18,45 @@ interface ProjectConfiguration {
 }
 
 export async function createApp(projectConfiguration: ProjectConfiguration) {
-    console.log(kleur.white(`Creating a new Comet app in `) + kleur.yellow(`${process.cwd()}\n`));
+    console.log(`Creating a new Comet app in ${kleur.blue(`${process.cwd()}\n`)}`);
     createWorkingDirectoryCopy(projectConfiguration.projectName, projectConfiguration.verbose);
-    cleanupReadme();
+    cleanupReadme(projectConfiguration.verbose);
```
Comment (johnnyomair) [inline create-app/src/scripts/create-app/createApp.ts:23]: Now that verbose is part of projectConfiguration, I think projectConfiguration isn't a good name anymore. Maybe use `options: CommandOptions` instead?

Comment (max-debug022) [reply create-app/src/scripts/create-app/createApp.ts]: Changes in: https://github.com/vivid-planet/comet-starter/pull/196/commits/fa381ca27b37c7a1689dde8701108d4ab1d1dd01

**Review by johnnyomair — COMMENTED** [review]
Please merge main and resolve conflicts. Thanks!

**Review by nsams — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Review by thomasdax98 — APPROVED** [review]

**Pull request merged**